modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -752,6 +755,34 @@ public void testIdleVerifyOnInactiveClusterWithPersistence() throws Exception {
         assertContains(log, testOut.toString(), "The check procedure has finished, no conflicts have been found.");
     }
 
+    /**
+     *
+     */
+    @Test
+    public void testIdleVerifyCancelCommand() throws Exception {
+        IgniteEx srv = startGrids(4);
+
+        srv.cluster().state(ACTIVE);
+
+        IgniteCache<Integer, Integer> cache =  srv.createCache(DEFAULT_CACHE_NAME);
+
+        for (int i = 0; i < 100000; i++)
+            cache.put(i, i);
+
+        new Thread(() -> execute("--cache", "idle_verify") ).start();
+
+        assertEquals(EXIT_CODE_OK, execute("--cache", "idle_verify", "--cancel"));
+
+        int idleVerifyCnt = 0;
+
+        for (ComputeTaskView view : srv.context().systemView().<ComputeTaskView>view(TASKS_VIEW)) {
```

Comment:
System view are local.
Let's iterate started servers and check them all.

Reply:
checked all servers